sdcm/utils/url_validate.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+import requests
+
+SUPPORTED_WEBSITES = ['s3.amazonaws.com', 'scratch.scylladb.com']
+URL_REGEX_AND_FORMAT_LIST = [
+    (re.compile(r'baseurl=(?P<url>http.*?)\$basearch/'), '{url}' + 'x86_64/repodata/repomd.xml'),
+    (re.compile(r'deb\s+\[arch=(?P<arch>.*?)\]\s(?P<url>http.*?)\s(?P<version_code_name>.*?)\s(?P<component>.*?)$'),
+     "{url}/dists/{version_code_name}/{component}/binary-{arch}/Packages"),
+    (re.compile(r'deb\s+(?P<url>http.*?)\s(?P<version_code_name>.*?)\s(?P<component>.*?)$'),
+     "{url}/dists/{version_code_name}/{component}/binary-amd64/Packages.gz"),
+]
+
+
+def verify_url_is_valid(url):
+    """
+    Check the URL of `scylla.list` is valid and verify it's a file.
+    After that check, the file contains a valid scheme and package URL.
+    """
+    supported_content_types, package_urls = ['application/octet-stream', 'binary/octet-stream'], []
+
+    if not url.startswith('http'):
+        url = f"{'http://'}{url}"
```

Comment:
what if someone wants to use FTP, NFS or even HTTPS??

Reply:
is this URL the one inside the .repo or .list files?

Reply:
@fgelcer 
Do you have the `FTP` or `NFS` URL that I can check?

Reply:
nope, but we can try to add them for future tests